sql-select/inactive-customers.sql

```diff
@@ -0,0 +1,3 @@
+SELECT *
+  from "customers"
+WHERE "isActive" = 'false';
```

Comment:
Make sure you're using the boolean `false`, not the string `'false'`:

---

![](https://user-images.githubusercontent.com/18158834/240113275-58cc03e4-8715-4321-a12e-f268b258d843.png)